script.js

```diff
@@ -1,4 +1,151 @@
+// there are 4 mode .
+// 1) player 1 roll . 2) player 1 pick to make the highest number.
+// 3) player 2 roll. 4) player 2 pick to make the highest number.
+var mode = 'player 1 roll dice';
+
+// store player 1 highest number
+var playerOneHighestNum = 0;
+
+var randomDice1 = 0;
+var randomDice2 = 0;
+
+// There are 2 dice.
+var diceRoll1 = function () {
+  var randomDiceNum = Math.floor(Math.random() * 6) + 1;
+  return randomDiceNum;
+};
+
+var diceRoll2 = function () {
+  var randomDiceNum2 = Math.floor(Math.random() * 6) + 1;
+  return randomDiceNum2;
+};
```

Comment:
This communicates to me that you don't understand functions and how they work. Why do we need two functions here that do the same thing?

Reply:
@Vincentworld1 this is a serious misunderstanding that we should address immediately